src/map-app/app/view/sidebar/directory.ts

```diff
@@ -365,17 +181,15 @@ export class DirectorySidebarView extends BaseSidebarView {
       .append("button")
       .attr("class", "w3-button w3-border-0 ml-auto sidebar-button")
       .attr("title", `${this.presenter.parent.mapui.labels.close} ${initiative.name}`)
-      .on("click", function () {
-        EventBus.Directory.initiativeClicked.pub(undefined);
-      })
+      .on("click", () => EventBus.Directory.initiativeClicked.pub(undefined))
       .append("i")
       .attr("class", "fa " + "fa-times");
     initiativeContentElement
       .append("div")
       .html(initiativeContent);
     initiativeSidebar.classed("sea-initiative-sidebar-open", true);
-    // if (document.getElementById("map-app-leaflet-map").clientWidth < 800)
-    if (window.outerWidth < 800)
+
+    if (getViewportWidth() <= 800)
```

Comment:
Another magic constant (not one of your creation of course, but same rationale as above applies, maybe)